distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Ref.enso

```diff
@@ -39,3 +39,19 @@ type Ref
              (Ref.new 0) . put 10
     put : Any -> Any
     put self new_value = @Builtin_Method "Ref.put"
+
+    ## Gets the contents of this mutable reference, transform it using the
+       provided function, and put the new value into the reference.
+
+       Returns the original value.
+
+       Arguments:
+       - fun: the function to transform the value in the ref
+
+        > Example
+          Increment the integer stored in a ref.
+
+          (Ref.new 10) . modify (_+1) . get
+          # => 11
```

Comment:
This example does not seem to be aligned with the contract of the method (saying that `modify` returns the 'original value').

I guess the example needs to be amended.

Ideally, we should have an analogous test verifying the example.

Reply:
Done